doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -304,14 +304,14 @@ PyTorch
 .. dropdown:: Log to W&B
 
     .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking//torch_exp_tracking_wandb.py
-            :emphasize-lines: 15, 16, 17, 21, 22, 51, 52, 54, 55
+            :emphasize-lines: 15-17, 19-21, 59-60, 62-63
```

Comment:
Can change this to just be the line of code.

```suggestion
            :emphasize-lines: 16, 19-21, 59-60, 62-63
```

<img width="686" height="165" alt="Image" src="https://github.com/user-attachments/assets/45ba4550-5133-432e-b3ed-7ba61ec38e3d" />